files/etc/workarea.env

```diff
@@ -0,0 +1,14 @@
+#
+# Workarea Application Configuration
+#
+# For more configuration settings, check out:
+# https://developer.workarea.com/articles/configuration-for-hosting.html
+
+RAILS_ENV="production"
+WORKAREA_MONGOID_HOST="localhost:27017"
```

Comment:
note for all you security-minded folks: both mongodb and elasticsearch are only configured to accept requests from localhost, not 127.0.0.1, so they shouldn't be accessible from outside this box.